src/apps/reservation-list/list/reservation-list.test.ts

```diff
@@ -304,11 +306,14 @@ describe("Reservation list", () => {
     }).as("digital_reservations");
 
     // Intercept covers.
-    cy.fixture("cover.json")
-      .then((result) => {
-        cy.intercept("GET", "**/covers**", result);
-      })
-      .as("cover");
+    cy.interceptGraphql({
+      operationName: "GetCoversByPids",
+      fixtureFilePath: "cover.json"
+    });
+    cy.interceptGraphql({
+      operationName: "GetBestRepresentationPidByIsbn",
+      fixtureFilePath: "cover-get-best-representation-by-isbn.json"
+    });
```

Comment:
:+1: We need to add this extra intercept here as we have a reservation which might not refer to a pid.